src/stream/src/executor/mview/materialize.rs

```diff
@@ -79,7 +81,13 @@ impl<S: StateStore> MaterializeExecutor<S> {
 
         let schema = input.schema().clone();
 
-        let state_table = StateTable::from_table_catalog(table_catalog, store, vnodes).await;
+        // TODO: If we do some `Delete` after schema change, we cannot ensure the encoded value
+        // with the new version of serializer is the same as the old one, even if they can be
+        // decoded into the same value. The table is now performing consistency check on the raw
+        // bytes, so we need to turn off the check here. We may turn it on if we can compare the
+        // decoded row.
+        let state_table =
+            StateTableInner::from_table_catalog_inconsistent_op(table_catalog, store, vnodes).await;
```

Comment:
cc @hzxa21 😰

Reply:
cc. @hzxa21 Sanity check is disabled here